pallets/funding/src/functions/6_settlement.rs

```diff
@@ -146,32 +167,31 @@ impl<T: Config> Pallet<T> {
 			Error::<T>::SettlementNotStarted
 		);
 
-		// Return either the full amount to refund if bid is rejected/project failed,
-		// or a partial amount when the wap > paid price/bid is partially accepted
+		// Return the full bid amount to refund if bid is rejected or project failed,
+		// Return a partial amount if the project succeeded, and the wap > paid price or bid is partially accepted
 		let BidRefund { final_ct_usd_price, final_ct_amount, refunded_plmc, refunded_funding_asset_amount } =
 			Self::calculate_refund(&bid, funding_success, wap)?;
 
-		Self::release_participation_bond(&bid.bidder, refunded_plmc)?;
+		if bid.mode == ParticipationMode::OTM {
+			<pallet_proxy_bonding::Pallet<T>>::refund_fee(
+				project_id,
+				HoldReason::Participation.into(),
+				&bid.bidder,
+				refunded_plmc,
+				bid.funding_asset.id(),
+			)?;
+		} else {
+			Self::release_participation_bond_for(&bid.bidder, refunded_plmc)?;
+		}
 		Self::release_funding_asset(project_id, &bid.bidder, refunded_funding_asset_amount, bid.funding_asset)?;
```

Comment:
Might be good to first do the release_funding_asset just in case the refund_fee < then min deposit amount. See previous pr for similar comment